src/components/CardsContainer.tsx

```diff
@@ -1,9 +1,31 @@
 'use client'
 
+import UserCard from './UserCard'
+import { namesArray } from '@/utils/sampleData'
+import { sealifeImages } from '@/utils/imageLists'
+
 export default function CardsContainer() {
+	const testNames = namesArray.slice(10, 30)
```

Comment:
nit: you forgot to rename this `testNames`

Reply:
DONE